api/.env

```diff
@@ -17,7 +17,7 @@ SHOW_GROUP_INVITATION_LINKS="true"
 # Task execution intervals in seconds
 CRON_CONVERT_IMAGES="60" # 1 minute
 CRON_CONVERT_VIDEO="3600" # 1 hour
-CRON_NOTIFICATION_EMAILS="3600" # 1 hour
+CRON_NOTIFICATION_EMAILS="0" # No restriction
```

Comment:
If there's no use to modify that parameter, we should maybe hardcode it. It could still live in service.yaml but at least not appear here, in the .env file

Reply:
A server admin could use it to prevent emails being sent so often. Say, you might be on a free tier for some email sending service, and don't want to go over your limit. You could set this to only allow email cron once per day to make sure each user only gets one summary email per day, preventing users from getting them more regularly. It wouldn't change the UI, but I could see some people perhaps using it.

Might be a confusion for upgrades though, if someone updates to the newest version and is confused as to why the option to send emails immediately isn't working because they have this set in their .env.

I'm happy with whatever you think is best.

Reply:
Maybe it should be renamed `NOTIFICATION_EMAILS_THROTTLING` then. I feel like it's okay to rename it as the next release is going to be a major one anw

Reply:
Yeah good idea. That solves both problems, lets admins use it still while also preventing them accidentally throttling emails when upgrading versions.

I've renamed it in a new commit.